### PR TITLE
Backup font when loRes font fails on parents and league pages

### PR DESCRIPTION
**Pull request by AndrewJakubowicz**
# Issue

Currently one of our heading fonts fails in production. The fallback font is a completely different style.

This change adds a fallback font that is in the spirit of the original pixel art font.

This change is best shown through screenshots. Below are screenshots of codecombat.com/parents and codecombat.com/league pages.

This is how the headings are supposed to appear:

image
image

The font loading may fail due to cors issues. Then the following is shown:

image
image

This PR adds this fallback:

image
image

# Testing

This can be tested manually in a local environment, running both `npm run dev` and `npm run proxy` in separate terminals.

Then navigate to localhost:3000/league and localhost:3000/parents

I find I can reproduce LoRes breaking on Chrome, whilst Safari seems to load the font.

### CODE REVIEW

File: app/core/initialize.coffee [hunk 1]
```diff
@@ -57,6 +57,8 @@ init = ->
   setUpBackboneMediator(app)
   app.initialize()
   loadOfflineFonts() unless app.isProduction()
+  # We always want to load this font.
+  $('head').prepend '<link rel="stylesheet" type="text/css" href="/fonts/vt323.css">'
```
Comment (nwinter) [inline app/core/initialize.coffee:61]: Is there a reason to load this here from JS rather than putting into our base static layout file?

**Comment by AndrewJakubowicz** [comment]
Doing a deploy to staging.codecombat.com for testing. Skipping code review as low code risk complexity. Instead want to check that font is loading on our staging infrastructure.

**Review by nwinter — APPROVED** [review]

File: app/core/initialize.coffee (hunk 1 above)
Comment (AndrewJakubowicz) [reply app/core/initialize.coffee:61]: I had initially reference the `loadOfflineFonts` method. However a better option is to add it to the head of only the two Vue pages that need it. Will do that and then trigger a deploy to staging.

Comment (AndrewJakubowicz) [reply app/core/initialize.coffee:61]: Punting this to next PR. Discussed at standup.

**Pull request merged**